src/admin/actions/UIActionsCreator.ts

```diff
@@ -35,3 +35,31 @@ export function modalWindowsChangeState(payload): IAction {
 		payload,
 	};
 }
+
+export function toggleMainPreloader(payload): IAction {
```

Comment:
I really don't like toggle for preloader. What if we call 2 functions that have to get it show?